src/org/thoughtcrime/securesms/reactions/ReactionRecipientsAdapter.java

```diff
@@ -0,0 +1,105 @@
+package org.thoughtcrime.securesms.reactions;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.recyclerview.widget.RecyclerView;
+
+import org.thoughtcrime.securesms.R;
+import org.thoughtcrime.securesms.mms.GlideRequests;
+import org.thoughtcrime.securesms.util.Pair;
+
+import java.util.ArrayList;
+
+public class ReactionRecipientsAdapter extends RecyclerView.Adapter
+{
+  @SuppressWarnings("unused")
+  private final static String TAG = ReactionRecipientsAdapter.class.getSimpleName();
+
+  private @NonNull ArrayList<Pair<Integer, String>> contactsReactions = new ArrayList<>();
+  private final LayoutInflater                li;
```

Comment:
while using short abbreviations if they're used only at a scope of few lines is okay (loops), for member variables, this is not-so-good style - eg. many will read this as "list item" at a first glance :)

better be verbose here and call the things what it is:


```suggestion
  private final LayoutInflater                layoutInflater;
```

(usages need to be adapted as well)